release-notes/EPM/I-DOCSIS/EPM_7.0.0_I-DOCSIS.md

```diff
@@ -0,0 +1,125 @@
+---
+uid: EPM_7.0.0_I-DOCSIS
+---
+
+# EPM 7.0.0 I-DOCSIS
+
+## New features
+
+## Changes
+
+### Enhancements
+
+#### CISCO CBR-8 CCAP Platform: Toggle button to enable/disable polling of OFDM/OFDMA tables [ID_36635]
+
+In some cases, it can occur that the CCAP is not able to retrieve OIDs when polling the OFDM/OFDMA tables, and DataMiner interprets this as a timeout.
+
+As such, it is now possible to enable or disable the polling of these tables with a new toggle button, so that you can prevent such timeouts.
+
+#### Only active channels associated with cable modem shown [ID_37376]
+
+When you view all channels associated with a cable modem, now only the active channels will be shown instead of all potentially available channels. This will make it easier to focus on the relevant cable modem KPIs.
+
+#### Visual pages now include parent shapes for enhanced network topology [ID_37592]
+
+Alignment errors on the Visual pages for Port, Distribution, and FAT (Forwarding Access Terminal) pages have been corrected. These Visual pages now include parent shapes, making it possible to identify the hub and the Optical Line Terminal (OLT) associated with each entity.
+
```

Comment:
Changed

Reply:
```suggestion
> [!NOTE]
> This update is related to the GPON solution and has been included in this package because of shared components between solutions.
```

Should this be included as a note, or is this internal?

Reply:
Hi @EdithLansens, please if we can include it as a note.
I am not sure if we need to rephrase it to make it more clear

Reply:
> [!NOTE]
> This change has been implemented in the EPM GPON Solution because of shared components across solutions.

I rephrased it. What do you think, @miguel-slc?

Reply:
Hi @EdithLansens ,
Although this change is related to the EPM GPON solution, it has been added to the EPM I-DOCSIS solution since both solutions currently share some components (Skyline EPM Platform connector)

Reply:
OK! Changing it to:
> [!NOTE]
> This change is related to the EPM GPON Solution because of shared components across solutions.

Reply:
Thanks. It looks ok!